docs/en/changelogs/2.3.0.md

```diff
@@ -47,18 +47,18 @@ If you **don't** require the functionality, it's safe to delete these database t
 #### auth_openid removed from default installation
 
 The auth_openid module has been removed from default installation.  Please install the module separately from
-[silverstripe.org](http://silverstripe.org/auth-openid-module/)
+`[silverstripe.org](http://silverstripe.org/auth-openid-module/)`
 
 ### GenericDataAdmin and RelatedDataEditor moved into new module
 
 GenericDataAdmin functionality has been moved into its own module called
 
-**[genericdataadmin](modules/genericdataadmin)**. If you were previously using this as a feature out of the box with
+**[genericdataadmin](https://github.com/silverstripe-labs/silverstripe-genericdataadmin)**. If you were previously using this as a feature out of the box with
 SilverStripe, then you will need to download this module to continue using it.
 
 ### User Defined Form moved into new module
 
-User Defined Form has been moved into its own module called **[userforms](modules/userforms)**. If you were previously
+User Defined Form has been moved into its own module called **[userforms](http://addons.silverstripe.org/add-ons/silverstripe/userforms)**. If you were previously
```

Comment:
Modules are all over the place. Sometimes in GH:silverstripe-labs, sometimes in GH:silverstripe, sometimes in addons, sometimes in personal repos....kinda chaotic. I think it would be good to make the submission, maintenance, documentation, storage, retrieval of modules more systematic...but first things first....